FirebaseAppCheck/Sources/Public/FirebaseAppCheck/FIRAppCheckDebugProvider.h

```diff
@@ -69,8 +73,8 @@ NS_SWIFT_NAME(AppCheckDebugProvider)
 
 /** Returns the currently used App Check debug token. The priority:
  *  - `FIRAAppCheckDebugToken` env variable value
- *  - previously generated stored local token
- *  - newly generated random token
+ *  - Previously generated stored local token
+ *  - Newly generated random token
```

Comment:
optional: Additional detail should be added here, something like "Newly generated random token. **The generated token will be stored locally for the future use**". I can add it in a follow up PR.